dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -680,9 +671,8 @@ protected String getCommitID(String repositoryId, Version version) {
     /**
      * Updates a user object with metadata from GitHub
      * @param user the user to be updated
-     * @return Updated user object
      */
-    public User getUserMetadata(User user) {
+    public void getUserMetadata(User user) {
```

Comment:
I guess since it doesn't return anything now it might be better named `setUserMetadata`?

Reply:
will do in a follow-up